### PR TITLE
feat: Initial page setting for custom book

### PR DESCRIPTION
**Pull request by Rafii2198**
Replaces contentBookShouldOpenWynntilsMenu setting with InitialPage enum, players can now choose which activity they want to open, by default it opens the main page as opening quests was confusing for new players (they didn't know there was more to the book) as well as Content Book having more activities than quests made it a bit tedious if you didn't want to focus on quests

### CODE REVIEW

**Review by magicus — APPROVED** [review]
This will make users lose their current config, but I think this sounds ok in this case.

**Comment by magicus** [comment]
Also, it would be great if we could have a keybind by default to open the Wynntils main menu (profile).

**Comment by Rafii2198** [comment]
On Discord we had multiple conversations about setting it false, this will essentially do that and give more possibilities

**Comment by Rafii2198** [comment]
We could change the K button which now opens Quests but as now there are more incentives to do other activities, changing it to main page would also make sense as it has most things that Content Book has, not only quests

**Comment by magicus** [comment]
No, please don't touch K! I still think quests is the most valuable part of the content book.

What about assigning J to open the profile page? It's next to K, and afaik it is not currently bound to anything.

**Comment by Rafii2198** [comment]
J is the guild map :( Maybe it could be . then? Or maybe Home button?

**Comment by magicus** [comment]
Maybe "i"?

**Comment by Rafii2198** [comment]
"i" could work. Should I change it in this pr or make a new one? Although I kinda like Home button as you know it is the home page of Wynntils, but it is your call.

**Comment by magicus** [comment]
I'm on a mac laptop. There is no home button. ;-) I think "i" as in "information" sounds like a good mnemonic. But yeah, it's probably better to make it in a separate PR, even if it is just a single line.

**Comment by magicus** [comment]
I'll wait for @kristofbolyai to approve that we skip the upfixer as well...

**Comment by ryanzhoudev** [comment]
> J is the guild map :( Maybe it could be . then? Or maybe Home button?

Unrelated, but why is the guild map not L? The default advancements map is L at the moment, but we have hybrid data parsing, what would someone use the vanilla map for?

File: common/src/main/java/com/wynntils/features/ui/WynntilsContentBookFeature.java
Comment (kristofbolyai) [inline common/src/main/java/com/wynntils/features/ui/WynntilsContentBookFeature.java:93]: But why did you change the default? By default, we used to open the quest book, now we open the user profile.

I don't mind not having the upfixer, but let's open the quest book by default.

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/features/ui/WynntilsContentBookFeature.java:93]: Okay, I see, it was confusing. Well, it is confusing either way..

**Comment by magicus** [comment]
I agree that it makes sense to replace the vanilla functionality, but I don't think we can change the vanilla keybinding, and the result would be that we have a keybinding conflict.

**Pull request merged**
